libnavigation-base/src/main/res/values-uz/strings.xml

```diff
@@ -2,7 +2,7 @@
 
     <!-- Time formatting -->
     <plurals name="mapbox_number_of_days">
-        <item quantity="other">kun</item>
```

Comment:
Looks like we need `other` as well, because I get the following crash, when the trip takes longer than one day:
```
android.content.res.Resources$NotFoundException: Plural resource ID #0x7f0d0002 quantity=8 item=other
    at android.content.res.ResourcesImpl.getQuantityText(ResourcesImpl.java:326)
    at android.content.res.Resources.getQuantityText(Resources.java:455)
    at android.content.res.Resources.getQuantityString(Resources.java:558)
    at com.mapbox.navigation.base.internal.time.TimeFormatter.formatDays(TimeFormatter.kt:108)
    at com.mapbox.navigation.base.internal.time.TimeFormatter.formatTimeRemaining(TimeFormatter.kt:94)
    at com.mapbox.navigation.ui.tripprogress.model.TimeRemainingFormatter.format(TimeRemainingFormatter.kt:31)
    at com.mapbox.navigation.ui.tripprogress.model.TimeRemainingFormatter.format(TimeRemainingFormatter.kt:17)
    at com.mapbox.navigation.ui.tripprogress.model.TripProgressUpdateFormatter.getTimeRemaining(TripProgressUpdateFormatter.kt:115)
    at com.mapbox.navigation.ui.tripprogress.view.MapboxTripProgressView.render(MapboxTripProgressView.kt:89)
    at com.mapbox.navigation.examples.core.MapboxNavigationActivity.routeProgressObserver$lambda-7(MapboxNavigationActivity.kt:229)
    at com.mapbox.navigation.examples.core.MapboxNavigationActivity.$r8$lambda$4EdaMIwhYKIMRWmmoWh2laPioJo(Unknown Source:0)
    at com.mapbox.navigation.examples.core.MapboxNavigationActivity$$ExternalSyntheticLambda3.onRouteProgressChanged(Unknown Source:2)
    at com.mapbox.navigation.core.trip.session.MapboxTripSession.updateRouteProgress(MapboxTripSession.kt:629)
    at com.mapbox.navigation.core.trip.session.MapboxTripSession.access$updateRouteProgress(MapboxTripSession.kt:58)
    at com.mapbox.navigation.core.trip.session.MapboxTripSession$navigatorObserver$1$onStatus$1.invokeSuspend(MapboxTripSession.kt:314)
    at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
    at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
    at android.os.Handler.handleCallback(Handler.java:883)
    at android.os.Handler.dispatchMessage(Handler.java:100)
    at android.os.Looper.loop(Looper.java:224)
    at android.app.ActivityThread.main(ActivityThread.java:7590)
    at java.lang.reflect.Method.invoke(Native Method)
    at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
    at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
```